modules/medianetBidAdapter.md

```diff
@@ -79,17 +79,16 @@ var videoAdUnit = {
   mediaTypes: {
     video: {
       context: "instream",
-      playerSize: [640, 480]
+      playerSize: [640, 480],
+      mimes: ['video/mp4'],
+      placement: 1
     }
   },
   bids: [{
     bidder: 'medianet',
     params: {
       cid: '8CUX0H51C',
-      video: {
-        mimes: ['video/mp4'],
-        placement: 1
```

Comment:
just move mimes, not placement, placement is a unique parameter to your bidder, not an openrtb parameter that should go to all bidders

Reply:
i was totally wrong, see 5.9 https://www.iab.com/wp-content/uploads/2016/03/OpenRTB-API-Specification-Version-2-5-FINAL.pdf